ProductAnalysis.py

```diff
@@ -21,24 +21,74 @@
 #######################################################################
 
 
-def createVocab(reviewDataList, stopWords):
-  # Fill in method here
-  pass
+def createVocab(reviewDataList, productList, stopWords):
+  print(reviewDataList, productList, stopWords)
+  # Iterate through all the json files data and create vocabulary dictionary having the words and their associated counts
+  # Use parseWords to generate the tokenized terms
+  # Use nltk.FreqDist to generate term frequqnecies
+  allTerms, reviewList, reviewFreqDictList, productIdList, reviewIdList, reviewContentList = [], [], [], [], [], []
+  for r in range(len(reviewDataList)):
+    for review in reviewDataList[r]:#['Reviews']:
+      parsedWords = parseWords(review['fullText'], stopWords)
+      reviewFrequency = dict(nltk.FreqDist(parsedWords))
+      reviewFreqDictList.append(reviewFrequency)
+      reviewList.append(parsedWords)
+      reviewIdList.append(review['reviewId'])
+      productIdList.append(productList[r])
+      reviewContentList.append(review['fullText'])
+      allTerms += parsedWords
+  termFrequency = nltk.FreqDist(allTerms)
+  vocab, cnt = [], []
+  vocabDict = {}
+  for k,v in termFrequency.items():
+    if v > 5:
```

Comment:
wheres magic number 5 come from?

Reply:
I kept it consistent with HotelAnalysis. @ssubramanian90 can yo help answer Jackson's questions?